content/_img-sources/cryptographic-keys.uxf

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<diagram program="umlet" version="14.2">
```

Comment:
This is the source file for `cryptographic-keys.svg`. I used [UMLet](https://umlet.com/) to make it. Reviewers don't need to review this file directly.